.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+### Summary
+
+Describe the changes made in this PR and why they are needed.
+
+### Testing
+
+Describe the testing done for this PR.
+
+### Applications affected
+
+List all known applications (UFS WM, JEDI, SRW, etc.) intentionally or unintentionally affected by this PR.
+
+### Systems affected
+
+List all systems intentionally or unintentionally affected by this PR.
+
+### Dependencies
+
+If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and link to the relevant issues/PRs.
```

Comment:
This label does not exist in the spack-stack repository. How about this:
```suggestion
If there are PRs that need to be merged before or along with this one, please add "- [ ] waiting on LINK_TO_PR" for each of them
```

Reply:
@AlexanderRichert-NOAA If you can make this one change, then we can merge this PR and be done with it. Thanks!